doc/dms_setup.md

```diff
@@ -0,0 +1,56 @@
+# Data Migration Service Setup
+
+## Overview
+
+There are a few parts to the setting up of DMS. The account running the DMS terraform needs to have certain AWS service access rights. Configure the VPC peering that is required for the DMS pipelining to other AWS accounts. Set the [create-cloudfoundry] pipeline against the appropriate environment. Make a note of any peering ids that are required. Setup the DMS configuration file named after the environment. Run the `create-cloudfoundry` job called `dms-terraform`.
+
+## Setup Steps
+
+1. Make sure that the concourse AWS account has enough rights to access DMS.
+1. There may be vpc peerings required. Ensure that they are setup before progressing further.
+1. Set the [create-cloudfoundry] pipeline against the appropriate environment. Make a note of the peering id(s) and the cidr block(s) associated with them.
```

Comment:
We also ought to assume that this document will be read at a time when this has all been merged to `main`, so setting the pipeline will be unnecessary (esp in production)

Reply:
👍